packages/purpose-template-process/test/api/getRiskAnalysisTemplateDocument.test.ts

```diff
@@ -0,0 +1,135 @@
+  purposeTemplateRiskAnalysisFormNotFound,
+  purposeTemplateRiskAnalysisTemplateDocumentNotFound,
+} from "../../src/model/domain/errors.js";
+
+const {
+  HTTP_STATUS_BAD_REQUEST,
+  HTTP_STATUS_FORBIDDEN,
+  HTTP_STATUS_OK,
+  HTTP_STATUS_NOT_FOUND,
+} = constants;
+
+describe("API GET /purposeTemplates/:purposeTemplateId/document test", () => {
+  const mockPurposeTemplate: PurposeTemplate = {
+    ...getMockPurposeTemplate(),
+  };
+
+  const mockRiskAnalysisDocument: RiskAnalysisTemplateDocument = {
+    id: generateId<RiskAnalysisTemplateDocumentId>(),
+    path: "s3://path/to/the/risk/analysis/file.pdf",
+    contentType: "application/pdf",
+    name: "risk_analysis_file.pdf",
+    prettyName: "Risk analysis document",
+    createdAt: new Date(),
+  };
+
+  beforeEach(() => {
+    purposeTemplateService.getRiskAnalysisTemplateDocument = vi
+      .fn()
+      .mockResolvedValue(mockRiskAnalysisDocument);
+  });
+
+  const makeRequest = async (
+    token: string,
+    purposeTemplateId: PurposeTemplateId = mockPurposeTemplate.id
+  ) =>
+    request(api)
+      .get(`/purposeTemplates/${purposeTemplateId}/document`)
+      .set("Authorization", `Bearer ${token}`)
+      .set("X-Correlation-Id", generateId())
+      .send();
+
+  const authorizedRoles: AuthRole[] = [
+    authRole.ADMIN_ROLE,
+    authRole.SUPPORT_ROLE,
+  ];
+
+  it.each(authorizedRoles)(
+    "Should return 200 for user with role %s on successful risk analysis template document retrieved",
+    async (role) => {
+      const token = generateToken(role);
+      const res = await makeRequest(token);
+
+      expect(res.status).toBe(HTTP_STATUS_OK);
+      expect(res.body).toEqual(
+        purposeTemplateApi.RiskAnalysisTemplateDocument.parse(
+          riskAnalysisTemplateDocumentToApiRiskAnalysisTemplateDocument(
+            mockRiskAnalysisDocument
+          )
+        )
+      );
+    }
+  );
+
+  it.each(
+    Object.values(authRole).filter((role) => !authorizedRoles.includes(role))
+  )("Should return 403 for user with role %s", async (role) => {
+    const token = generateToken(role);
+    const res = await makeRequest(token);
+    expect(res.status).toBe(HTTP_STATUS_FORBIDDEN);
+  });
+
+  it.each([
+    {
+      error: purposeTemplateNotFound(mockPurposeTemplate.id),
+      expectedStatus: HTTP_STATUS_NOT_FOUND,
+      description: "purposeTemplateNotFound",
+    },
+    {
+      error: purposeTemplateRiskAnalysisFormNotFound(
+        mockPurposeTemplate.id,
+        generateId<RiskAnalysisFormTemplateId>()
+      ),
+      expectedStatus: HTTP_STATUS_NOT_FOUND,
+      description: "purposeTemplateRiskAnalysisFormNotFound",
+    },
+    {
+      error: purposeTemplateRiskAnalysisTemplateDocumentNotFound(
+        generateId<RiskAnalysisFormTemplateId>()
+      ),
+      expectedStatus: HTTP_STATUS_NOT_FOUND,
+      description: "purposeTemplateRiskAnalysisTemplateDocumentNotFound",
+    },
+  ])(
+    "Should return $expectedStatus for $description error",
```

Comment:
We can just use `$error.code` instead of `description`
```suggestion
    "Should return $expectedStatus for $$error.code error",
```